orchestrator/test_runner/Cargo.toml

```diff
@@ -38,3 +38,4 @@ rand = "0.8"
 tonic = "0.6"
 futures = "0.3"
 serde_json = "1.0"
+sha2 = "0.10"
```

Comment:
Maybe I'm missing something but I believe this is unnecessary

Reply:
I used it at once point, good catch.